spec/controllers/project_comments_controller_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ProjectCommentsController, type: :controller do
+  login_user
+
+  let(:user) { User.first }
```

Comment:
`User.first` actually works here bc you have `current_user` but it is not intuitive what `User.first` is. So change it to `subject.current_user`.

Reply:
:+1: